client/subcomponents/volunteers/VolunteerListItem/volunteerListItemHelper.js

```diff
@@ -3,18 +3,17 @@ import { dispatch, getState } from '../../../store'
 import { setWaiting, clearWaiting } from '../../../slices/waiting'
 import { showError } from '../../../slices/error'
 
-export function toggleAttendance(volunteerData, consume = requestor) {
-  const storeState = getState()
-  const { token } = storeState.user
-  dispatch(setWaiting())
+export async function toggleAttendance(volunteerData, consume = requestor) {
+  try {
```

Comment:
Sweet, same as the other file:  move the try down, to above line 12

Reply:
Thanks! Have fixed it now, let us know if its all good :)